dist/pinny.js

```diff
@@ -100,49 +100,54 @@
          */
         animation: {
             openComplete: function() {
-                var plugin = this;
-                this._disableExternalInputs();
-                this._focus();
+                setTimeout(function () {
```

Comment:
@donnielrt 

Nvm. Using RAF causes the mocha tests to fail :(. This seems good enough :+1: